pos_customer_wallet/static/src/js/screens.js

```diff
@@ -55,11 +55,19 @@ odoo.define("pos_customer_wallet.screens", function (require) {
                     return false;
                 }
                 if (product_lines_qty > 0) {
+                    var wallet_product_names = [];
+                    var wallet_products = this.find_customer_wallet_products();
+                    wallet_products.forEach(function (product) {
+                        wallet_product_names.push(product.display_name);
+                    });
                     this.gui.show_popup("error", {
                         title: _t("No customer selected"),
-                        body: _t(
-                            "Cannot use customer wallet Product without selecting a customer.\n\n Please select a customer or use a different payment method."
-                        ),
+                        body:
+                            _t("Cannot sell the product '") +
+                            wallet_product_names.join(",") +
+                            _t(
+                                "' without selecting a customer. Please select a customer or remove the order line(s)."
+                            ),
```

Comment:
yes, :+1: In fact, it's just because I don't how to to do it, but i'll investigate.

Reply:
Hi. I investigated without success.
- I didn't found the correct way to do it.
- Odoo use "+" syntax. see : https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/static/src/js/screens.js#L2118
- same in OCA/pos repo.

Do you know how to do it ? 

Otherwise, propose to ignore that theoritical-for-the-time-being problem.